cli_tools/gce_vm_image_import/importer/args.go

```diff
@@ -203,6 +205,9 @@ func (args *ImportArguments) registerFlags(flagSet *flag.FlagSet) {
 	flagSet.BoolVar(&args.NoExternalIP, "no_external_ip", false,
 		"VPC doesn't allow external IPs.")
 
+	flagSet.BoolVar(&args.Inspect, "inspect", false,
+		"Run experimental disk inspections.")
```

Comment:
Minor: remove `experimental`. It's likely this will be forgotten once it's not experimental any more and the description will be out of date.

Reply:
sgtm; updated